src/framing.c

```diff
@@ -1492,6 +1492,22 @@ const int head3_7[] = {0x4f,0x67,0x67,0x53,0,0x05,
                        1,
                        0};
 
+int compare_packet(const ogg_packet *op1, const ogg_packet *op2){
+  if (op1->packet!=op2->packet)
+    return(1);
```

Comment:
Please wrap the conditional body in braces. No one wants to repeat Apple's double goto TLS bug.

I think it would be useful to `fprintf(stderr, ...);` in each return clause about what the mismatch was, so the error trace saves someone a debugging step.

Note that local style has no spaces around the keywords. ```if(op1->packet!=op2->packet){```